docs/reference/aq-json-export-format.md

```diff
@@ -0,0 +1,187 @@
+| snapshot.date                           | Date/time in ISO 8601 format, corresponding to the version of the code that was analyzed.                       |                       
+| systemElement.id                        | ID based on the system element's fingerprint. System elements will retain the same ID across subsequent scans.  |
+| systemElement.name                      | Full/long name. Not unqiue.                                                                                     |
+| systemElement.shortName                 | Short/display name. Not unique. Optional field, use `name` when not specified                                   |
+| systemElement.type                      | See the [list of system element types](#system-element-types) below.                                            |
+| systemElement.measurementValues         | Maps metric names to numerical metric values, for metrics based on source code analysis.                        |
+| systemElement.measurementTimeSeries     | Maps metric names to time series, for metrics based on change history.                                          |
+| systemElement.observationType           | See the list of [architecture observations](#architecture-observations) below. Optional field. [1]              |
+| systemElement.technology                | Indicates this is a technology-specific system element. Optional field. [2]                                     |
+| systemElement.annotation                | Text description that provides information for system elements. Optional field, defined in configuration. [3]   |
+| dependency.sourceElementId              | Refers to one of the entries in `systemElements`.                                                               |
+| dependency.targetElementId              | Refers to one of the entries in `systemElements`.                                                               |
+| dependency.type                         | See the [list of dependency types](#dependency-types) below.                                                    |
+| dependency.description                  | Textual description of the dependency sub-type. Unlike `type`, this is purely a text label and *not* an enum.   |
+| dependency.count                        | Describes the number of dependencies. Optional, assume 1 when not specified.                                    |
+| dependency.undesirable                  | Dependency is considered undesirable as specified in the scope file. Optional field, defaults to false.         |
+| technology.name                         | Technology display name.                                                                                        |
+| technology.contextName                  | SAT context name. Note that this field is not available for technologies not supported by the SAT.              |
+
```

Comment:
Should we add a note that the JSON contains more fields, but that any undocumented fields should not be used as we do not consider them to be stable?